pulser-core/pulser/noise_model.py

```diff
@@ -136,6 +139,13 @@ class NoiseModel:
     - **doppler**: Local atom detuning due to termal motion of the
       atoms and Doppler effect with respect to laser frequency.
       Parametrized by the ``temperature`` field.
+    - **register**: Thermal fluctuations in the
+      register positions, parametrized by ``temperature``, ``trap_waist``
+      and, ``trap_depth``, which must all be defined,
+      - Plane standard deviation: 𝜎ˣʸ = √(T w²/(4 Uₜᵣₐₚ)), where T is
+      temperature, w is the trap waist and Uₜᵣₐₚ is the trap depth.
+      - Off plane standard deviation: 𝜎ᶻ = 𝜋 / 𝜆 √2 w 𝜎ˣʸ, where 𝜆 is the
+      trap wavelength with a constant value of 0.85 µm
```

Comment:
I let you check it [in the documentation](https://pulser--915.org.readthedocs.build/en/915/noise_model.html), but it's not rendering great. May I suggest
```suggestion
    - **register**: Thermal fluctuations in the
      register positions, parametrized by ``temperature``, ``trap_waist``
      and, ``trap_depth``, which must all be defined,

      - Plane standard deviation: :math:`\sigma^{xy} = \sqrt{\frac{T w²}{4 U_{trap}}}`, where T is
      temperature, w is the trap waist and :math:`U_{trap}` is the trap depth.
      - Off plane standard deviation: :math:`\sigma^z = \frac{\pi}{\lambda \sqrt{2} w \sigma^{xy}}`, where :math:`lambda` is the
      trap wavelength with a constant value of 0.85 µm
```

Reply:
(see my suggestion above ;) )